docs/pages/components/radio-group.md

```diff
@@ -74,19 +100,50 @@ Add descriptive help text to a radio group with the `help-text` attribute. For h
 ```
 
 ```pug:slim
-sl-radio-group label="What would you like to do?" name="a" value="issue_shares" help-text="Contact support if you don't see the option you need here"
+sl-radio-group[
+  label="What would you like to do?"
+  name="a"
+  value="issue_shares"
+  help-text="Contact support if you don't see the option you need here"
+]
   sl-radio value="issue_shares" Issue shares
   sl-radio value="employee_buyback" Employee buyback
   sl-radio value="cancel_certificate" Cancel a certificate
 br
 br
-sl-radio-group label="What would you like to do?" name="a" value="issue_shares"
+sl-radio-group[
+  label="What would you like to do?"
+  name="a"
+  value="issue_shares"
+]
   sl-radio value="issue_shares" Issue shares
   sl-radio value="employee_buyback" Employee buyback
   sl-radio value="cancel_certificate" Cancel a certificate
   div slot="help-text"
     a href="#" class="ts-text-link" Contact support
     | if you don't see the option you need here
+
+/*
+  When rendering with ts_form_for
+  — NOTE: To set default value for initial page load, ensure a value is set
+  in the controller's #new action:
+  e.g. @cap_table_event = CapTableEvent.new(a: "issue_shares")
```

Comment:
```suggestion
  e.g. if using `ts_form_for @cap_table_event`, set @cap_table_event = CapTableEvent.new(a: "issue_shares")
```

Reply:
TY! Updated all the places where this comment appears!